pkg/plugin/plugin.go

```diff
@@ -77,52 +83,63 @@ func NewJiraPlugin(ctx context.Context, cfg *PluginConfig) (*JiraPlugin, error)
 // Validate returns the validation result.
 func (j *JiraPlugin) Validate(ctx context.Context, req *jvspb.ValidateJustificationRequest) (*jvspb.ValidateJustificationResponse, error) {
 	if got, want := req.Justification.Category, jiraCategory; got != want {
-		return nil,
-			fmt.Errorf("failed to perform validation, expected category %q to be %q",
-				got, want)
+		err := fmt.Errorf("failed to perform validation, expected category %q to be %q", got, want)
+		log.Printf("failed to validate jira justification: %v", err)
```

Comment:
Could you elaborate why we can't use pkg/logging? There seems no obvious blocker to me

Reply:
Logs aren't printed to the cloud logging when I use `pkg/logging`. I came across the plugin doc https://github.com/hashicorp/go-plugin#features, it only mentions the standard go log and hashcrop log lib. So I'm guessing the go-plugin can't capture the log printed by `pkg/logging`

Reply:
Good to know. I will take a look

Reply:
Could you share how did you use pkg/logging? I'm pretty sure it by default logs to stdout and should be streamed back

Reply:
It's already used at https://github.com/abcxyz/jvs-plugin-jira/blob/main/pkg/cli/server.go#L87C28-L87C48 
But this log statement is not available in cloud logging https://screenshot.googleplex.com/KCvYwKZmng4B2kD

Reply:
After offline discussion, logging is skipped and marked with TODO comments for further investigation